src/main/java/org/exschool/pocketworld/model/BildingResource.java

```diff
@@ -0,0 +1,51 @@
+package org.exschool.pocketworld.model;
+
+
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Entity;
+
```

Comment:
BuildingResource should contain such fields: 
int amount (amount of resource)
int level (level of building)
ResourceType type (enum that represents type of resource - wood, rock, etc)
BuildingType type (enum that represent type o building)